Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs

```diff
@@ -25,7 +25,13 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 		void SetupLayer()
 		{
-			Layer.CornerRadius = 5;
+			float cornerRadius = Element.CornerRadius;
+
+			if (cornerRadius == -1f)
+				cornerRadius = 5f; // default corner radius
```

Comment:
@activa Equality comparison is guaranteed to work for floating point numbers when the numbers are power of two. This is not an issue here.